django3_codemods/config_tools/tox_modernizer.py

```diff
@@ -0,0 +1,75 @@
+import sys
+import re
+from configparser import ConfigParser, NoSectionError
+
+TOX_SECTION = "tox"
+ENV_SECTION = "envlist"
+TEST_ENV_SECTION = "testenv"
+TEST_ENV_DEPS = "deps"
+PY_38 = "py{38}"
+DJANGO_22 = "django{22}"
```

Comment:
For tox, we should go ahead and add Django 3.0 testing now.  But wait to add it in `.travis.yml` until we've addressed basic things like relevant deprecation warnings.